README.rst

```diff
@@ -40,12 +40,18 @@ Contributing
 
 See `CONTRIBUTING.rst`_ for more details on developing for the Zigzag project.
 
+Release Process
+---------------
+
+See `docs/release_process.rst`_ for information on the release process for 'zigzag'
```

Comment:
Not rendering correctly because you've got `docs/release_process.rst_` instead of `release_process.rst_`.